meson.build

```diff
@@ -4,8 +4,9 @@ project('KNLMeansCL',
         license : 'GPL-3',
         version : '1.1.1')
 
+avisynth = dependency('avisynth')
 vapoursynth = dependency('vapoursynth')
-deps = [vapoursynth]
```

Comment:
Why are you hardcoding Avisynth dependency? It's not in my distro and I don't feel like I need it either.

Reply:
Oversight, but since the AviSynth plugin sources are always built [even if not effective], the srcs list needs to be amended too.